src/openapi.yaml

```diff
@@ -171,21 +171,19 @@ paths:
       parameters:
       - name: X-NLX-Request-Application-Id
         in: header
-        description: Identificatie van de applicatie die het verzoek stuurt (indien
-          NLX wordt gebruikt).
+        description: Application that performs request
```

Comment:
this translation reset shouldn't happen - do you have a language code override in your local settings?

Reply:
I don't have language local settings in django projects. This description is the same as in vng_api_common.inspectors ([link](https://github.com/VNG-Realisatie/vng-api-common/blob/5e7f452ce98f0d9950d57252b7e5d5cc5fc66c98/vng_api_common/inspectors/view.py#L77))
Should it be translated to Dutch?

Reply:
Oh, I think it was because I generated schema using local vng_api_common. I'll fix it